packages/node-app/src/models/giveaway.model.ts

```diff
@@ -0,0 +1,35 @@
+import mongoose, { Schema } from 'mongoose';
+
+enum Unit {
+  NODE = 'node',
+  RAILS = 'rails',
+  PYTHON = 'python'
+}
+
+export interface Requirements {
+  unit?: Unit;
+}
+
+interface Giveaway extends mongoose.Document {
+  title: string;
+  description: string;
+  startTime: Date;
+  endTime: Date;
+  participants: string[];
+  winners: string[];
```

Comment:
I think we should also have a user schema, so we can map the email with the name, role, profile image, etc

Reply:
We will get the user information from an external database so the only thing we need to save on our side is a list of the account addresses that will be used to identify the users